tools/cleaner.ts

```diff
@@ -32,9 +35,14 @@ async function main() {
       console.error(`Cannot save followup: ${followupAnonymized.id}`)
       console.trace(err)
     }
+    console.log(["Terminé", "Followup", followup_count].join(";"))
   }
 
-  console.log(["Terminé", "Followup", followup_count].join(";"))
+  const deleteSixMonthsOldFollowups = await Followup.deleteMany({
+    createdAt: { $lt: sixMonthsAgo },
+  })
+
+  console.log(`Deleted ${deleteSixMonthsOldFollowups.deletedCount}`)
```

Comment:
ici on peut ajouter un peu de context dans le log car si on a "Deleted 63" on sait pas trop de quoi on parle (enfin il faut regarder le code pour comprendre. 

```suggestion
  console.log(`Deleted ${deleteSixMonthsOldFollowups.deletedCount} followups`)
```